src/github.com/getlantern/flashlight/client/chained.go

```diff
@@ -25,82 +22,50 @@ var ForceChainedProxyAddr string
 // ForceAuthToken - If specified, auth token will be forced to this
 var ForceAuthToken string
 
-// ChainedServerInfo provides identity information for a chained server.
 type ChainedServerInfo struct {
 	// Addr: the host:port of the upstream proxy server
 	Addr string
 
-	// Pipelined: If true, requests to the chained server will be pipelined
-	Pipelined bool
-
 	// Cert: optional PEM encoded certificate for the server. If specified,
 	// server will be dialed using TLS over tcp. Otherwise, server will be
-	// dialed using plain tcp.
+	// dialed using plain tcp. For OBFS4 proxies, this is the Base64-encoded obfs4
+	// certificate.
 	Cert string
 
 	// AuthToken: the authtoken to present to the upstream server.
 	AuthToken string
 
-	// Weight: relative weight versus other servers (for round-robin)
-	Weight int
-
-	// QOS: relative quality of service offered. Should be >= 0, with higher
-	// values indicating higher QOS.
-	QOS int
-
 	// Trusted: Determines if a host can be trusted with plain HTTP traffic.
 	Trusted bool
+
+	// PluggableTransport: If specified, a pluggable transport will be used
+	PluggableTransport string
+
+	// PluggableTransportSettings: Settings for pluggable transport
+	PluggableTransportSettings map[string]string
 }
 
 // Dialer creates a *balancer.Dialer backed by a chained server.
```

Comment:
Minor nit, and this is only related to this PR in the sense that with pluggable transports there's more for the code reader to think about, but I think a comment for deviceID would be helpful here @oxtoacart -- i.e. make it clear it's not related to pluggable transports at all but just identifies this device to the server.